packages/api-docs-builder-core/baseUi/migratedBaseComponents.ts

```diff
@@ -0,0 +1,23 @@
+export const migratedBaseComponents = [
```

Comment:
I lack a bit of context to know what are those migrated components 🙈

Reply:
Yeah, me too, TBH. I haven't analyzed it deeply yet. It surely can be done better :)